### PR TITLE
Add backfilling action example

### LINKED ISSUE #941: [Docs RFC] Add scheduling backfill to user-defined actions examples
# Describe change in content, appearance, or functionality

Ryan provided [an example of a user-defined action to schedule backfill](https://www.timescale.com/forum/t/schedule-backfill/290/4). This would be useful to have in our UDA examples.

# Subject matter expert (SME)

@ryanbooz 

# Deadline

[When does this need to be addressed]

# Any further info

[Anything else you want to add, or further links]

### PR DESCRIPTION
**Pull request by charislam**
# Description

Add example of scheduled backfill to user-defined actions.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #941

### CODE REVIEW

File: timescaledb/how-to-guides/user-defined-actions/example-backfill.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Schedule backfill
+Create a procedure to regularly backfill data into a compressed hypertable.
+Requires the [`decompress_backfill`][decompress_backfill] function from the
+TimescaleDB extras repository.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/example-backfill.md:4]: Do we need a little more exposition here? It feels a little bare ...

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
@charislam What is the status of this, since we've just merged UDA How To updates?

**Comment by charislam** [comment]
> @charislam What is the status of this, since we've just merged UDA How To updates?

Needs a rewrite to fit the template of the other Example pages! Doesn't conflict though as it's an entirely new page and different example from the rest. I'll put it on the to-do list and mark as draft for now

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
@charislam I think this one is good to merge, when you're ready

**Pull request merged**